README.md

```diff
@@ -39,6 +39,28 @@ jobs:
         branch: ${{ github.ref }}
 ```
 
+An example workflow to use the branch parameter to push the changes to specified branch e.g. an PR related branch:
+
+```yaml
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+        with:
+          ref: ${{ github.head_ref }}
+          fetch-depth: 0
+    - name: Commit files
+      run: |
+        git config --local user.email "github-actions[bot]@users.noreply.github.com"
+        git config --local user.name "github-actions[bot]"
+        git commit -m "Add changes" -a
```

Comment:
```suggestion
        git commit -a -m "Add changes"
```
I suggest adding options that don't take any argument before the ones that do.

Reply:
Done